<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackUnpackWindow</class>
 <widget class="QWidget" name="PackUnpackWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cbPackAlgo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RLE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LZ77</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Huffman</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lePath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Choose path to save...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAction">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChoosePath">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>90</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
